### DISCUSSION TITLE
Welcome to CNPS-Rust Discussions!

### DISCUSSION THREAD

**Discussion by AsteroidusTv** [AsteroidusTv/CNPS-Rust#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Vital-Vuillaume** [comment]
The project is very good I congratulate you

**Reply by AsteroidusTv** [reply]
Hello !

**Comment by AsteroidusTv** [comment]
Your discution is absolutly useless !!!
